Source/Core/Ellipsoid.js

```diff
@@ -27,10 +29,13 @@ define([
         z = defaultValue(z, 0.0);
 
         //>>includeStart('debug', pragmas.debug);
-        if (x < 0.0 || y < 0.0 || z < 0.0) {
+        /*if (x < 0.0 || y < 0.0 || z < 0.0) {
             throw new DeveloperError('All radii components must be greater than or equal to zero.');
-        }
+        }*/
```

Comment:
Since this PR was submitted alongside https://github.com/AnalyticalGraphicsInc/cesium/pull/5424 it has some of the same issues. Do a pass over the file to make sure commented code is deleted, checks are within `includeStart/includeEnd`, and the appropriate `Check.typeOf` is used.